client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -34,6 +34,8 @@ interface KeyboardShortcut {
 	ariaLabel: string;
 }
 
+const USER_STORE = User.register();
```

Comment:
Nit: maybe move this so it's right beside the `DOMAIN_SUGGESTIONS_STORE` declaration?